src/Package/Impl/Expansions/ExpansionClient.cs

```diff
@@ -284,5 +234,53 @@ public sealed class ExpansionClient : IVsExpansionClient {
             return VSConstants.S_OK;
         }
         #endregion
+
+        private static TextSpan TextSpanFromSpan(ITextBuffer textBuffer, Span span) {
+            var ts = new TextSpan();
+            ITextSnapshotLine line = textBuffer.CurrentSnapshot.GetLineFromPosition(span.Start);
+            ts.iStartLine = line.LineNumber;
+            ts.iEndLine = line.LineNumber;
```

Comment:
In this particular code yes. However, I agree this is confusing. Let me make it generic.